README.md

```diff
@@ -11,6 +13,7 @@ for how to install Foreman plugins
 
 | Foreman Version | Katello Version | Plugin Version |
 | --------------- | --------------- | -------------- |
```

Comment:
I think we should bump the minor version (1.7.0) for adding a new foreman katello version to the matrix. It makes maintenance easier, and allows for bugfix releases.